tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -152,10 +152,22 @@ In Godot's **Editor → Editor Settings** menu:
 
 - Set **Mono** -> **Editor** -> **External Editor** to **Visual Studio**.
 
-Next, you need to download the Godot Visual Studio extension from github
+Next, you can download the Godot Visual Studio extension from github
 `here <https://github.com/godotengine/godot-csharp-visualstudio/releases>`__.
 Double click on the downloaded file and follow the installation process.
 
+.. note:: As of Godot 3.2.3, the Godot Visual Studio extension is not fully
```

Comment:
At this point, if this bug has been around for so long, we should just state that it's not fully compatible without mentioning the Godot version. I don't think it's helpful to suggest users go back to Godot 3.2.1 to fix this.

Reply:
Ah, I somewhat mindlessly used this phrase just because other 3.2.3 stuff on the page is using it, and the correlation to the new csproj format is vaguely interesting to me. 😄 I didn't think about how it comes across as a recommendation to downgrade. Definitely agree on that point, will remove.